documentation/modules/exploit/multi/http/apache_druid_cve_2023_25194.md

```diff
@@ -0,0 +1,76 @@
+## Vulnerable Application
+
+Apache Kafka is an open-source distributed event streaming platform that is used for real-time data streaming and processing.
+Kafka clients are a set of Java libraries that allow you to produce and consume messages from Apache Kafka.
+
+In the version prior to 3.3.2, there is a JNDI injection issue in the Apache Kafka clients if an attacker is able to set the
+`sasl.jaas.config` property for any of the connector's Kafka clients to `com.sun.security.auth.module.JndiLoginModule`.
+It will allow the server to connect to the attacker's LDAP server and deserialize the LDAP response, which the attacker can
+use to execute java deserialization gadget chains on the Kafka connect server. Attacker can cause unrestricted deserialization
+of untrusted data (or) RCE vulnerability when there are gadgets in the classpath.
+
+As Apache Druid dependeds on kafka-clients to connect to one of its datasources, making it out of the box vulnerable if using any
+affected version.
+
+The below docker compose file can be used to build a vulnerable container.
+
+```doocker-compose
```

Comment:
This seems unnecessary as docker-compose is not the language (syntax highlighting won't be applied).
```suggestion
```
```